### PR TITLE
Add a new blogs page and v2.3.0 release

### PR DESCRIPTION
**Pull request by arcturus2**
* Minor update to order of content on the home page
* Updated the links to the download page to go directly to the download
  page
* Added a blogs page
* Added the first blog which is the release of v2.3.0

### CODE REVIEW

**Comment by arcturus2** [comment]
Attached is the first preview based on the commit so far. I will be working on the rest of the images now. If you download and extract the zip file you should be able to click on the index.html page and have a look.

[site-v3.zip](https://github.com/constellation-app/constellation-app.github.io/files/6572943/site-v3.zip)

**Comment by arcturus2** [comment]
Attached is the latest version including the pending images and some feedback.

[site-v4.zip](https://github.com/constellation-app/constellation-app.github.io/files/6573486/site-v4.zip)

File: docs/download.html [hunk 1]
```diff
@@ -73,9 +74,9 @@ <h3>Download Constellation</h3>
                                 <li>Constellation Adapters</li>
```
Comment (antares1470) [inline docs/download.html]: Adaptors

**Comment by aldebaran30701** [comment]
Love the look of the blog! Loads of progress.
One thing I found a bit odd was the fact that you click 'read more' to view the rest of the blog. It may be useful for when many blogs are listed one under another - but it just felt confusing for me. Totally okay to keep it as is, just a small mention of it.
Couldn't see anything to pick on it - gave the whole blog page a good read.

**Comment by sol695510** [comment]
image

I think it looks great for the most part, but I think that the title here needs to be more prominent, since the background has a lot going on (which I think is fine). As labelled in the screenshot:
1. I think this title could just be made bigger.
2. This line/divider could be made "bold" because it blends into the background a bit as it is.
3. This could be made bigger and possibly "bold" as well, but up to you since its a subtitle.

As it is, this component takes up about a 1/3 of this section of background, whereas I think it should be between 1/2 to 2/3 to increase its readability.

**Comment by sol695510** [comment]
Current:
placeholder

Proposed:
Recent_Updates_Logo_Idea

This is just a rough design, but I think the Recent Updates carousel would look better with some version text on the icon for each linked version, not too much text but some key text for quick visual recognition.

File: docs/download.html (hunk 1 above)
Comment (arcturus2) [reply docs/download.html]: Fixed thank you very much for picking up on the typo

**Comment by arcturus2** [comment]
> I think it looks great for the most part, but I think that the title here needs to be more prominent, since the background has a lot going on (which I think is fine). As labelled in the screenshot:
> 
>     1. I think this title could just be made bigger.
> 
>     2. This line/divider could be made "bold" because it blends into the background a bit as it is.
> 
>     3. This could be made bigger and possibly "bold" as well, but up to you since its a subtitle.
> 
> 
> As it is, this component takes up about a 1/3 of this section of background, whereas I think it should be between 1/2 to 2/3 to increase its readability.

Yeah that is decent feedback. I have not been able to make this change today but will consider doing so tomorrow

**Comment by arcturus2** [comment]
> This is just a rough design, but I think the Recent Updates carousel would look better with some version text on the icon for each linked version, not too much text but some key text for quick visual recognition.

Totally agree with this so for now I've removed the images all together to stop the repetition. I was looking at an alternate design by putting the version number at an angle to one of the corners but started spending too much time I stopped for now.

**Review by aldebaran30701 — APPROVED** [review]

**Review by Delphinus8821 — APPROVED** [review]

**Comment by arcturus2** [comment]
The latest site with some suggestions made. Others will be pushed back to next time.

[site-v5.zip](https://github.com/constellation-app/constellation-app.github.io/files/6581139/site-v5.zip)

**Comment by arcturus2** [comment]
@sol695510 i created https://github.com/constellation-app/constellation/issues/1078 to track your suggestions for next time. Running out of time to fix them now.

**Pull request merged**